### DISCUSSION TITLE
Possible placement of theme.json outside the theme?

### DISCUSSION THREAD

**Discussion by AlkoKod** [WordPress/gutenberg#44532] [Q&A]
Hi, I just start working with Gutenberg Theme.json and I am curious if I can place it somewhere on FTP. I can't place it to the theme folder because I am using the Oxygen builder which completely disable the themes. 

Any idea where to store the theme.json file and how to force WordPress to load it on INIT? Curently, I have it in wp-content/uploads/blocks/js/theme.json 

I will be happy for any ideas!

**Comment by t-hamano** [comment]
Hi.
How about using the `wp_theme_json_data_theme` hook introduced in WordPress 6.1 to overwrite it with the `theme.json` loaded from the specified directory?

[Filters for theme.json data – Make WordPress Core](https://make.wordpress.org/core/2022/10/10/filters-for-theme-json-data/)
